source/Octopus.Cli/Commands/Releases/CreateReleaseCommand.cs

```diff
@@ -79,6 +79,9 @@ public class CreateReleaseCommand : DeploymentCommandBase, ISupportFormattedOutp
 
         protected override async Task ValidateParameters()
         {
+            if (!string.IsNullOrWhiteSpace(VersionNumber) && (!SemanticVersion.TryParseStrict(VersionNumber, out var result) || result is null)) 
+                throw new CommandException($"The release number '{VersionNumber}' does not appear to be a valid version number. The version must consist of between 1 and 4 number only parts before the optional pre-release part. e.g. 2, 2.1, 2.4.0.23, 2.4-beta and 1-beta.2");
```

Comment:
Took this message from the `ReleaseNumberValidationRule` in Octopus Server

Reply:
The web portal (and backend) will quite happily let you create a release with a textual version number.
E.g. I just created one where the release version was "donkey" and it works fine.

Is there anything wrong with that? Do we need to restrict it further?

Reply:
You're right. I confused myself with this one and was only thinking in package versions 😫

Going back to the drawing board.